src/main/java/com/imsweb/datagenerator/naaccr/NaaccrDataGenerator.java

```diff
@@ -270,16 +274,23 @@ public NaaccrDataGeneratorRule getTumorRule(String ruleId) {
         if (options == null)
             options = new NaaccrDataGeneratorOptions();
 
+        // Context contains information that is shared amongst the rules. Contains values we want to use for this patient.
+        Map<String, String> context = generateInitialPatientContext(numTumors);
```

Comment:
Is there a specific reason this is String/String as opposed to String/Object? I see for example you put a tumor index in there, but you translate the index into a string. Wouldn't it be better to leave it as an Integer?

Reply:
Fixed.